pkg/controllers/state/suite_test.go

```diff
@@ -1431,3 +1433,132 @@ func ExpectStateNodeNotFoundForMachine(machine *v1alpha5.Machine) *state.Node {
 	ExpectWithOffset(1, ret).To(BeNil())
 	return ret
 }
+
+var _ = Describe("DaemonSet Controller", func() {
```

Comment:
Probably want to put the `Describe` blocks together so the file is organized as

1. Tests
2. Helper Functions